### DISCUSSION TITLE
What is the maximum number of vector dimensions supported by qdrant?

### DISCUSSION THREAD

**Discussion by kdh1122** [qdrant/qdrant#4519] [Q&A] [answered]
I wanted to look for it in the official documentation, but I couldn't find it.

**Comment by xrd** [comment]
I spoke to someone from Qdrant at the ai.engineer conference today and I think they said 64k is the limit.

**Reply by timvisee** [reply, marked as answer]
The current limit for dense vectors [is](https://github.com/qdrant/qdrant/blob/5c29cad76edc4fe33882d385c02633352666d67e/lib/collection/src/operations/types.rs#L1224-L1225) 65536, though this _can_ be lifted if there is a valid use case for higher dimensionality.

For sparse vectors the current limit is 4294967295 dimensions.

Note that hardware is the biggest limitation here.
